README.md

```diff
@@ -48,13 +48,19 @@ Run the service:
 $ flask run --debugger --port 8080
 ```
 
-Open a new terminal. Apply `imagemanifestvuln` example data:
+Open a new terminal. Get a token:
+
+```
+$ curl localhost:8080/api/v1/token?scope=imagemanifestvuln
+```
+
+Apply `imagemanifestvuln` example data:
 
 ```
 $ curl --request POST \
 --header "Content-Type: application/json" \
 --data @examples/imagemanifestvuln.json \
-localhost:8080/api/v1/imagemanifestvuln/app-sre-prod-01
+localhost:8080/api/v1/imagemanifestvuln/app-sre-prod-01?token=<token>
```

Comment:
Ohh, I missed it in my review... I believe the token should come in the `Authorization` header, not as an URL param

Reply:
I like the idea of the token being in the headers. But why in the `Authorization` one?

Reply:
The token is used for that, right? In order to POST data, you need a valid token. Withouth token, you should get a 401. Expired token, you should get a 401. That's all specified in teh rfc2626, section 14.8:

https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html

Maybe I'm too attached to the docker API implementation. Do you have some other header in mind?

Reply:
ok, I think I understand it. The token actually authorizes you to write your data.